color_refinement.py

```diff
@@ -221,18 +226,21 @@ def get_number_automorphisms(g: Graph) -> int:
 
     graphs = L[0]
     print("Graph: ", GRAPH)
+    automorphisms = {}
```

Comment:
Ik zou dit stuk in een andere PR doen.

Reply:
Oeps, dat is per ongeluk in deze terecht gekomen volgens mij :) Staat al in de #54 PR